llvm/utils/lit/lit/main.py

```diff
@@ -124,7 +124,7 @@ def main(builtin_params={}):
     run_tests(selected_tests, lit_config, opts, len(discovered_tests))
     elapsed = time.time() - start
 
-    if not opts.skip_test_recording or opts.time_tests:
+    if not opts.skip_test_time_recording or opts.time_tests:
```

Comment:
Should time-tests really overwrite skip-test-time-recording here?

Reply:
The original idea was to prevent the case where both `--skip-test-time-recording` and `--time-tests` are both passed. If it didn't overwrite, then we'd run into the situation where we wouldn't time the tests, but attempt to print the histogram.

Now that I think about it, the `--time-tests` parameter is redundant since you cannot enable both `--time-tests` and `--skip-test-time-recording` as they are mutually exclusive args.